ce/ce/archivers/ZipUnpacker.ts

```diff
@@ -49,6 +49,19 @@ export class ZipUnpacker extends Unpacker {
 
       const openedFile = await archiveUri.openFile();
       const zipFile = await ZipFile.read(openedFile);
+      if (options.strip === -1) {
```

Comment:
It seems like we should do this for all extractors that support "strip", not just zip.

Reply:
I could add that for tar based ones, but it would require going thru the whole file twice - the filenames in a tar archive are spread out the whole way.  (The zip implementation gets them up front regardless).

I guess it only has to do it if `-1 ` is specified. 

So the question is how bad do we want that? Lemme see how fast I can do that.

Reply:
Can we just make it something other than "strip" then? Like "remove-prefixes" or something, so that what users see has the same behavior when it's the same name. (Or just make the tar extractor explicitly say "I can't do that" when a user does)

Reply:
(That is to say: if we are saying "strip means this" it should mean that everywhere and emit diagnostics if that's impossible)

Reply:
Updated tar to match zip extractor behavior for guessing the folders to strip